src/System.Runtime.CompilerServices.Unsafe/ref/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard1.0;
+      netstandard;
```

Comment:
> catches exactly the type of cycle you suggest

Awesome! Thanks

> If you'd prefer to undo the factoring 

This factoring is good because of it gives us flexibility. My concern was only about what kind of build we get for netcoreapp.

> we can cross-compile for NETCoreApp and have it reference either System.Runtime, or even System.Private.Corelib

Agree. We will want to do that as we try to do https://github.com/dotnet/corefx/pull/15768 again. cc @ahsonkhan @mellinoe